src/routes/reference/bufnmfcross.svx

```diff
@@ -9,4 +9,73 @@ tags:
     - spectral
 flair: reference
 category: Transform
----
\ No newline at end of file
+---
+
+<script>
+    import IndentNote from '$lib/components/IndentNote.svelte';
+    import Image from '$lib/components/Image.svelte';
+    import Audio from '$lib/components/Audio.svelte';
+    import ResourceLink from '$lib/components/ResourceLink.svelte';
+</script>
+
+BufNMFCross uses non-negative matrix factorisation ("NMF", for more info, see [BufNMF](/reference/bufnmf)) to reconstruct the components of a ``target`` sound using spectral templates of a ``source`` sound. This process is a type of audio mosaiking (or "musaiking") that is intended to convey certain aspects of a target recording (such as melody and rhythm) using sound components (such as timbre) from a source recording. The result is a hybrid sound whose character depends on how well the target can be represented by the source's spectral templates. 
+
+### Target: Drum Loop
+
+<Image
+src="/reference/bufnmfcross/target.jpg"
+label=""
+/>
+
+<Audio
+src="/reference/bufnmfcross/target.mp3"
+label="Drum loop used as a target."
+waveform={true}
+/>
+
+### Source: Synthesizer sounds 
+
+<Image
+src="/reference/bufnmfcross/source.jpg"
+label=""
+/>
+
+
+<Audio
+src="/reference/bufnmfcross/source.mp3"
+label="Synth sounds used as the source."
+waveform={true}
+/>
+
+### Output:
+
+<Image
+src="/reference/bufnmfcross/output.jpg"
+label=""
+/>
+
+<Audio
+src="/reference/bufnmfcross/output.mp3"
+label="Output of BufNMFCross using the drums as the target and the synth sounds as the source."
+waveform={true}
+/>
+
+Rather than replacing single spectral frames in the target with single spectral frames in the source, BufNMFCross looks for opportunities to use a sequence of spectral frames from the source buffer to enhance the perception of the timbral morphology of both the source and target. This functionality can be adjusted using the ``continuity`` argument.
+
+Additionally, BufNMFCross will avoid repeating a spectral frame within a specified duration of time (using the argument ``timeSparsity``). This helps avoid a common problem with this type of audio mosaiking: the repetition or over use of a single sound element from the source.
```

Comment:
over use -> overuse

one could talk of a similar approach to round-robin in modern samplers?

Reply:
> one could talk of a similar approach to round-robin in modern samplers? 
nice one. added that in.